docs/bug-bash/README.md

```diff
@@ -0,0 +1,146 @@
+# Azure MCP Server - Bug Bash Welcome Guide
+
+Welcome to the Azure MCP Server Bug Bash! We're excited to have you help us improve the quality and reliability of Azure MCP Server across different platforms and scenarios.
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Bug Bash Goals](#bug-bash-goals)
+- [What to Test](#what-to-test)
+- [How to Report Issues](#how-to-report-issues)
+- [Testing Scenarios](#testing-scenarios)
+- [Resources](#resources)
+
+## Introduction
+
+The Azure MCP Server enables AI agents to interact with Azure services through natural language commands. As we continue to enhance the server, we need your help to identify issues across different platforms, IDEs, and usage scenarios.
+
+This bug bash focuses on:
+- **Multi-platform compatibility** (Windows, macOS, Linux)
+- **Installation and setup** across different IDEs
+- **Resource discovery and inspection** - testing how well MCP finds and retrieves information
+- **Querying capabilities** - testing database queries and data retrieval
+- **Monitoring and diagnostics** - testing log/metric access and health checks
+- **Deployment guidance** - testing how MCP helps plan and guide deployments
+- **Authentication** across different environments
+- **End-to-end scenarios** that developers commonly encounter
+
+## Bug Bash Goals
+
+The primary goals of this bug bash are to:
+
+1. **Validate cross-platform compatibility** - Ensure the server works reliably on Windows, macOS, and Linux
+2. **Verify installation experience** - Test installation across VS Code, Visual Studio, and IntelliJ IDEA
+3. **Assess performance** - Monitor memory consumption and CPU usage under typical workloads
+4. **Validate authentication** - Ensure auth works consistently across all platforms
+5. **Test server modes** - Verify single, namespace, and all modes work as expected
+6. **Validate feature flags** - Test enabling/disabling server features
+7. **Exercise real-world scenarios** - Run through common developer workflows
+
+## What to Test
+
+We encourage you to test the following areas:
+
+### Platform Testing
+- [ ] **Windows** - Test on Windows 10/11
+- [ ] **macOS** - Test on macOS (Intel and Apple Silicon)
+- [ ] **Linux** - Test on Ubuntu, Fedora, or other distributions
+
+### IDE Installation
+- [ ] **VS Code** - Stable and Insiders versions
+- [ ] **Visual Studio 2022** - Community, Professional, or Enterprise
+- [ ] **IntelliJ IDEA** - Ultimate or Community editions
+
+### Performance Monitoring
+- [ ] Monitor **memory consumption** during typical operations
```

Comment:
tell them how they should monitor (e.g. Task Manager on Windows)

Reply:
added